Modules/_uuidmodule.c

```diff
@@ -4,17 +4,10 @@
 #include <uuid/uuid.h>
 
 
-/* bpo-11063: libuuid on macOS doesn't provide uuid_generate_time_safe(),
-   only uuid_generate_time(). */
-#ifndef __APPLE__
-#  define HAVE_TIME_SAFE
-#endif
-
-
 static PyObject *
 py_uuid_generate_time_safe(void)
 {
-#ifdef HAVE_TIME_SAFE
+#if defined(HAVE_UUID_GENERATE_TIME_SAFE) && HAVE_UUID_GENERATE_TIME_SAFE
```

Comment:
I don't think that it's correct to test the define *value*. Only check if it's defined:

#ifdef HAVE_UUID_GENERATE_TIME_SAFE

Reply:
Yeah, I was about to write about that. I couldn't make autoconf set `#define HAVE_UUID_GENERATE_TIME_SAFE 0` if `uuid_generate_time_safe()` isn't available yet. Working on it :)

Reply:
What was broken about the original patch except that some autogenerated comments were suboptimal?

The existing `AC_CHECK_LIB(dld, shl_load)` should have all the same problems.